datastore/src/main/java/io/spine/server/storage/datastore/DsProperties.java

```diff
@@ -49,6 +60,22 @@ static void addAggregateIdProperty(Object aggregateId, Entity.Builder entity) {
         entity.set(aggregate_id.toString(), propertyValue);
     }
 
+    static void addCreatedProperty(Timestamp when, Entity.Builder entity) {
+        final long millis = Timestamps.toMillis(when);
+        final Date date = new Date(millis);
+        final DateTime dateTime = DateTime.copyFrom(date);
+        entity.set(AggregateEventRecordProperty.created.toString(), dateTime);
```

Comment:
We need to be consistent in naming enums. Some of them are lowercase, some of them are uppercase. I think it would be better to have them uppercase with proper `toString()` and static import of `AggregateEventRecordProperty`.  

You can also have a static method that would accept 1) `Entity.Builder` 2) property instance (as an interface) and 3) property value (as a generified argument). And the method would call `toString()`.